src/content/blog/2024_AmbassadorBG.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Unitary Fund Announces Brian Goldsmith as Newest Ambassador
+author: Kallie Ferguson
+day: 19
+month: 1
+year: 2024
+---
+
+# Unitary Fund Announces Brian Goldsmith as Newest Ambassador
+
+![Photo of Brian Goldsmith](/images/bdg_cropped.jpg)
+
+## Meet Brian
+
+_Q: How did you learn about Unitary Fund?_
+
+Brian: During a series of workshops put on by Zapata Computing, error mitigation was explored in which we used Mitiq for ZNE. Later when looking to engage in an open source project, I looked closer into Mitiq and found the Unitary Fund.
+
+_Q: Tell the community a bit about yourself. For example, how you got into quantum and your current interests._
+
+Brian: After 14 years as a software support engineer at IBM, I went back to school to get a Master's in Computer Science from Vanderbilt University. During my studies, I took an Introduction to Quantum Computing which sparked my interest in the field. I have enjoyed each aspect of quantum computing that I have been exposed to including error mitigation, simulation, and resource estimation. Outside of quantum computing, I am married to a high school science teacher and we have two boys and a dog.
+
+_Q: What's the first thing you did within the Unitary Fund community? What's the thing you're most proud of that you've been involved in?_
+
+Brian: After lurking for some time, I opened and worked on a PR related to installing Mitiq with z-shell. Since the PR consisted of adding quotes to a single line of documentation, it was a gentle introduction to contributing. I am quite proud of recent work done simplifying the expectation_estimation_shadow function. Though there were some complications, I enjoyed the challenge and appreciated the assistance and guidance from the team (including Andrea, Min, and Nate.)
```

Comment:
```suggestion
Brian: After lurking for some time, I opened and worked on a pull request (PR) related to installing Mitiq with z-shell. Since the PR consisted of adding quotes to a single line of documentation, it was a gentle introduction to contributing. I am quite proud of recent work done simplifying the [`expectation_estimation_shadow`](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.shadows.classical_postprocessing.expectation_estimation_shadow) function in the [*classical shadows*](https://mitiq.readthedocs.io/en/stable/guide/shadows.html) protocol in Mitiq. The protocol is based on the idea of shadow tomography, which is a technique for reconstructing a quantum state from a small number of measurements. Though there were some complications, I enjoyed the challenge and appreciated the assistance and guidance from the team (including Andrea, Min, and Nate.)
```